tests/Contracts/KubeServiceBase.test.ts

```diff
@@ -41,4 +41,12 @@ describe("KubeServiceBase Tests", () => {
             expect(output).toBe(KubeResourceType.Services);
         });
     });
+
+    it("getPods calls with labelSelector as input",() => {
+        expect.assertions(1);
+        const labelSelector:string = "app=app";
+        return service.getPods(labelSelector).then(output =>{

```

Comment:
we can add a case for labelSelector also by maksing changes to MockKubeService class